listening-comp/backend/tts_service.py

```
@@ -0,0 +1,397 @@
+                model="gpt-4",
+                messages=[{
+                    "role": "system",
+                    "content": "You are a Japanese pitch accent analyzer. For the given text, provide the pitch accent pattern for each word in JSON format. Include mora-by-mora breakdown and accent position."
+                }, {
+                    "role": "user",
+                    "content": f"Analyze pitch accent for: {text}"
+                }],
+                response_format={"type": "json_object"}
+            )
+            
+            pitch_data = json.loads(response.choices[0].message.content)
```

Ruff (0.8.2)

Undefined name `json`

(F821)